blueocean-material-icons/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@jenkins-cd/blueocean-material-icons",
+  "version": "0.0.4",
+  "description": "Material icons for the Jenkins Design Language",
+  "main": "dist/js/components/index.js",
+  "scripts": {
+    "lint": "gulp lint",
```

Comment:
you pretty much just want a `prepublish` that is something like `babel -src src -dst dist` or whatever, don't think it's necessary to have anything complicated here